components/Navigation/Navigation.js

```diff
@@ -20,7 +20,7 @@ const Navigation = ({ items, defaultItem, router, me, ...props }) => {
   // TODO next js get the location is different?
   // const activeItem = router && router.pathname ? router.pathname.slice(1) : defaultItem
   const activeItem = router.pathname.slice(1)
-
+  console.log(me)
```

Comment:
Console.Log(Me) could be removed? :)

Reply:
Thanks Monique! I removed console.log(me) and moved examples folder out of Actions folder and back into Components.